src/app/contact/page.tsx

```diff
@@ -0,0 +1,177 @@
+        },
+        () => {
+          toast.error('Something went wrong!', {
+            theme: 'dark',
+          });
+        }
+      );
+  };
+
+  return (
+    <>
+      <div className='w-full h-full px-6 md:px-0'>
+        <div className='container flex items-center justify-center h-full mx-auto text-center md:py-4 xl:text-left'>
+          <div className='flex flex-col w-full max-w-[700px]'>
+            <motion.h2
+              variants={fadeIn('up', 0.2)}
+              initial='hidden'
+              animate='show'
+              exit='hidden'
+              className='mb-12 text-center text-[32px] md:text-[48px] leading-tight font-semibold'
+            >
+              Let's <span className='text-[#1E50FF]'>connect.</span>
+            </motion.h2>
+
+            <motion.form
+              ref={ref}
+              variants={fadeIn('up', 0.4)}
+              initial='hidden'
+              animate='show'
+              exit='hidden'
+              onSubmit={handleSubmit}
+              className='flex flex-col flex-1 w-full gap-6 mx-auto'
+            >
+              <div className='flex w-full gap-x-6'>
+                <input
+                  type='text'
+                  name='user_name'
+                  placeholder='Name'
+                  className='h-[52px] pl-6 rounded-lg w-full bg-transparent border outline-none focus:ring-1 focus:ring-[#1E50FF] border-white/20 placeholder:text-white/30 placeholder:font-light'
+                />
+                <input
+                  type='email'
+                  name='user_email'
+                  placeholder='Email'
+                  className='h-[52px] pl-6 rounded-lg w-full bg-transparent border outline-none focus:ring-1 focus:ring-[#1E50FF] border-white/20 placeholder:text-white/30 placeholder:font-light'
+                />
+              </div>
+              <input
+                type='text'
+                name='subject'
+                placeholder='Subject'
+                className='h-[52px] pl-6 rounded-lg w-full bg-transparent border outline-none focus:ring-1 focus:ring-[#1E50FF] border-white/20 placeholder:text-white/30 placeholder:font-light'
+              />
+              <textarea
+                name='message'
+                cols={30}
+                rows={10}
+                placeholder='Message'
+                className='h-[180px] resize-none p-6 rounded-lg w-full bg-transparent border outline-none focus:ring-1 focus:ring-[#1E50FF] border-white/20 placeholder:text-white/30 placeholder:font-light'
+              ></textarea>
+              <button
+                type='submit'
+                className='border rounded-full h-[52px] border-white/50 max-w-[170px] px-8 transition-all duration-300 flex items-center justify-center overflow-hidden hover:border-[#1E50FF] group'
+              >
+                <span className='group-hover:-translate-y-[120%] group-hover:opacity-0 transition-all duration-500'>
+                  Let's talk
```

Comment:
120:22  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities